next/components/pages/BasicDocumentPage.tsx

```diff
@@ -45,12 +47,28 @@ const CustomPageBreadcrumbs = ({ basicDocument }: IProps) => {
 }
 
 const BasicDocumentPage = ({ basicDocument }: IProps) => {
-  const { t } = useTranslation('common')
+  const { t, i18n } = useTranslation('common')
+  const { getDownloadAriaLabel } = useDownloadAriaLabel()
 
   const [expandDescription, setExpandDescription] = React.useState(false)
   const description = basicDocument?.attributes?.description
   const showExpandButton = description ? description.length > DESCRIPTION_LIMIT : false
 
+  let fileSize
+  if (basicDocument.attributes?.attachment?.data?.attributes?.size !== undefined) {
```

Comment:
There is few things to change and improve on this part of code:

First, when accessing nested entity, it is useful and cleaner to extract them or their properties 
```
const { attachment } = basicDocument.attributes ?? {}
```
or even
```
const { size } = basicDocument.attributes.attachment.data.attributes ?? {}
```

or both
```
const { attachment } = basicDocument.attributes ?? {}
const { size } = attachment.data.attributes ?? {}
```

Following this approach, you can tidy up the file a little bit by extracting also `title`, `file_category` etc.

Reply:
Prepnem asi do slovenciny :D a skratim si premenne podla komentara vyssie

```
if (size !== undefined)
```
Tu by som kludne napisal iba `if (size)`. Chapem, ze `size` je `number`, a teda moze byt aj nula, a to neprejde podmienkou, ale v skutocnosti je to asi jedno, lebo ak pride size rovan nule, tak je nieco zle. Ak by bolo velmi treba rozlisit `undefined` od nuly, tak by som pouzil nasu utils funkciu `isDefined(size)`, ktora kontroluju ci je nieco `null` alebo `undefined` namiesto porovnavania `!== undefined`.